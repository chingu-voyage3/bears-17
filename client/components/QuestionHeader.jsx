import React from 'react';
import PropTypes from 'prop-types';

const QuestionHeader = props => (
  <div className="page__header">
    <h2 className="page__title">{props.title}</h2>
  </div>
);

QuestionHeader.defaultProps = {
  title: '',
};

QuestionHeader.propTypes = {
  title: PropTypes.string,
};

export default QuestionHeader;
