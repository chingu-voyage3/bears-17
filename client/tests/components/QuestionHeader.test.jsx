import React from 'react';
import renderer from 'react-test-renderer';

import QuestionHeader from 'Components/QuestionHeader';

describe('QuestionHeader', () => {
  it('should match the snapshot', () => {
    const header = renderer.create(<QuestionHeader title="Test Title" />);

    expect(header.toJSON()).toMatchSnapshot();
  });
});
