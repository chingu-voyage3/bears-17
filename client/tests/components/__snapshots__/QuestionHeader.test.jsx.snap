// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionHeader should match the snapshot 1`] = `
<div
  className="page__header"
>
  <h2
    className="page__title"
  >
    Test Title
  </h2>
</div>
`;
